topics/rails/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-aliases: rails5, rails-application, ruby-on-rails, rails4, rubyonrails, rails-tutorial
+aliases: rails7, rails6, rails-application, ruby-on-rails, rubyonrails, rails-tutorial
```

Comment:
I think it would be okay to leave rails4 and rails5 here, and just add the newer versions 🤔  what do you think?

Reply:
@octokatherine OKay, let me keep `rails4`/`rails5` in the updated changeset.